fortytools.yaml

```diff
@@ -106,7 +106,62 @@ paths:
           description: The app identified by the corresponding query parameter is not installed for any client of the user
         403:
           description: Invalid credentials, there is no User with this email / password combination.
-
+  /auth/firebase:
+    post:
+      tags:
+      - Auth
+      description: |
+        Performs a login to get firebase credentials
+      security: []
+      requestBody:
+        description: the firebase credentials you want to post
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                email:
```

Comment:
Can you also please describe which parameters are required? I think all of them are https://swagger.io/docs/specification/describing-parameters/

Reply:
updated